internal/core/src/common/Json.h

```diff
@@ -91,36 +102,33 @@ class Json {
             parser.iterate(data_, data_.size() + simdjson::SIMDJSON_PADDING)
                 .get(doc);
         AssertInfo(err == simdjson::SUCCESS,
-                   fmt::format("failed to parse the json: {}", err));
+                   fmt::format("failed to parse the json {}: {}",
+                               data_,
+                               simdjson::error_message(err)));
         return doc;
     }
 
-    simdjson::ondemand::value
-    operator[](const std::string_view field) const {
-        simdjson::ondemand::value result;
-        auto err = doc().get_value()[field].get(result);
-        AssertInfo(
-            err == simdjson::SUCCESS,
-            fmt::format("failed to access the field {}: {}", field, err));
-        return result;
+    bool
+    exist(std::vector<std::string> nested_path) const {
+        std::for_each(
+            nested_path.begin(), nested_path.end(), [](std::string& key) {
+                boost::replace_all(key, "~", "~0");
```

Comment:
leave the comment here, make sure code reviewer understand it's the requirement of simdJson